scripts/run-sonar.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Could you please explain why you created this script ? For the Travis CI process ?
If I'm correct, it is never called...